apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Version>1.0.0</Version>
-    <TargetFrameworks>netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>netstandard1.5;net461</TargetFrameworks>
```

Comment:
Any reason for using 461 here rather than net45 as we do for everything else?

Reply:
The minimum library we can target is 451 based on the aspnetcore libs we depend on and AsyncLocalis not available in 451.

Reply:
Okay. It looks like net46 should be okay - maybe that would be better? Might as well go as low as we can, to support as many users as possible.